solutions/src/database/db_connection.js

```diff
@@ -0,0 +1,21 @@
+// Add code below to connect to your database
+const { Pool } = require('pg');
+const url = require('url');
+require('env2')('./config.env');
+
+if(!process.env.DB_URL) throw new Error('Enviroment variable DB_URL must be set');
+
+const params = url.parse(process.env.DB_URL);
+const [username, password] = params.auth.split(':');
+
+const options = {
+  host: params.hostname,
+  port: params.port,
+  database: params.pathname.split('/')[1],
+  max: process.env.DB_MAX_CONNECTIONS || 2,
+  user: username,
+  password: password,
+  ssl: (options.host !== 'localhost'),
```

Comment:
I get a reference error for options not being defined, you can use `this.host` instead or `params.hostname`